spec/lib/cms/layout_builder_spec.rb

```diff
@@ -0,0 +1,92 @@
+RSpec.describe Cms::LayoutBuilder do
+  let!(:english_site) { FactoryGirl.create(:site, is_mirrored: true) }
+  let!(:welsh_site)   { FactoryGirl.create(:site, :welsh, is_mirrored: true) }
```

Comment:
The factory girl is included in the specs and it using only the create call as default. I would remove the namespace to follow the pattern. Wdyt?